recipes/zfp/all/conanfile.py

```diff
@@ -119,6 +119,9 @@ def package_info(self):
             self.cpp_info.components["_zfp"].sharedlinkflags = openmp_flags
             self.cpp_info.components["_zfp"].exelinkflags = openmp_flags
 
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs.append("m")
```

Comment:
there are components, why does it work?

Reply:
@SpaceIm 
I forgot it.
Thank you for your review.

Reply:
...still a good question